homeassistant/components/cloud/tts.py

```diff
@@ -266,12 +414,10 @@ async def async_get_tts_audio(
         assert self.hass is not None
         gender: Gender | str | None = options.get(ATTR_GENDER)
         gender = handle_deprecated_gender(self.hass, gender)
-        original_voice: str | None = options.get(ATTR_VOICE)
-        if original_voice is None and language == self._language:
-            original_voice = self._voice
```

Comment:
We still want users to be able to set the default voice from the cloud preferences. `self._voice` is now no longer used. That doesn't seem right.